servers/xr/xr_hand_tracker.cpp

```diff
@@ -110,48 +103,11 @@ void XRHandTracker::set_tracker_type(XRServer::TrackerType p_type) {
 }
 
 void XRHandTracker::set_tracker_hand(const XRPositionalTracker::TrackerHand p_hand) {
-	ERR_FAIL_INDEX(p_hand, TRACKER_HAND_MAX);
-
-	switch (p_hand) {
-		case TRACKER_HAND_LEFT:
-			tracker_hand = TRACKER_HAND_LEFT;
-			hand = HAND_LEFT;
-			break;
-
-		case TRACKER_HAND_RIGHT:
-			tracker_hand = TRACKER_HAND_RIGHT;
-			hand = HAND_RIGHT;
-			break;
-
-		case TRACKER_HAND_UNKNOWN:
-		default:
-			ERR_FAIL_MSG("XRHandTracker must specify hand");
-			break;
+	if (p_hand != TRACKER_HAND_LEFT && p_hand != TRACKER_HAND_RIGHT) {
+		ERR_FAIL_MSG("XRHandTracker must specify hand");
 	}
-}
-
-void XRHandTracker::set_hand(XRHandTracker::Hand p_hand) {
-	ERR_FAIL_INDEX(p_hand, HAND_MAX);
-
-	switch (p_hand) {
-		case HAND_LEFT:
-			tracker_hand = TRACKER_HAND_LEFT;
-			hand = HAND_LEFT;
-			break;
-
-		case HAND_RIGHT:
-			tracker_hand = TRACKER_HAND_RIGHT;
-			hand = HAND_RIGHT;
-			break;
-
-		default:
-			ERR_FAIL_MSG("XRHandTracker must specify hand");
-			break;
-	}
-}
 
-XRHandTracker::Hand XRHandTracker::get_hand() const {
-	return hand;
+	tracker_hand = TRACKER_HAND_LEFT;
```

Comment:
Shouldn't this be:

```suggestion
	tracker_hand = p_hand;
```

Reply:
Oh, and since unknown isn't a valid value on a hand tracker, should the constructor set `tracker_hand = TRACKER_HAND_LEFT` as the default?

Reply:
I would say so

Reply:
Good catch - I must have been low on caffeine at that point